### ISSUE TITLE
Mixed colours for gym routes

### ISSUE THREAD

**Issue by birgander2** [theCrag/website#1846]
Bicoloured gym holds are quite common, but cannot be entered (white-green, black-orange, blue-yellow, etc.).

**Comment by brendanheywood** [comment]
Yup we're aware of this, I'd like to go one step further and just allow a picture of the texture, so you can do stuff like 'swirly grey on red' or 'wooden holds' etc. All work on gym specific stuff is kinda dependant on us getting some gyms on board to work closely with - are you involved in that gym you've created or just a regular?

**Comment by birgander2** [comment]
Only regular. What you propose is of course the golden solution. Maybe a very simple idea: Instead of showing the colour / texture, just allow alternatively showing the text only, i.e. "wooden", "'red / black", if the colour is not recognised automatically. Also, the grades are sometimes hard to read with dark coloured background - putting a box with the colour / text beside the grade appears better to me.

**Comment by birgander2** [comment]
Cool, but I cannot find out how to add a bicolour route that!? Selecting two colours doesn't work.

**Comment by brendanheywood** [comment]
This is for paying gyms and we setup a custom palette of colors which can be essentially anything, eg a pattern
